test/image-blueprints/layer2-presubmit/group1/rhel92-source.toml

```diff
@@ -21,6 +21,10 @@ version = "{{ .Env.SOURCE_VERSION }}"
 name = "microshift-selinux"
 version = "{{ .Env.SOURCE_VERSION }}"
 
+[[packages]]
+name = "microshift-olm"
+version = "{{ .Env.SOURCE_VERSION }}"
```

Comment:
Since this RPM will end up in most of the tests, we need to fix microshift-cleanup-data to include the OLM namespace.